performance/README.md

```diff
@@ -0,0 +1,27 @@
+# How to run the fallout tests
+
+## Setup Graphite
```

Comment:
Out of curiosity what is the benefit of having a separate graphite server? Maybe we should add a note explaining why we do this?

Reply:
The benefit is that they have different lifecycles and backends (DSE servers)
But it may be possible to achieve the same thing using one `yaml` - we can investigate that in a separate ticket.